### PR TITLE
feat: add Markdown support to proposal body

### PR DESCRIPTION
**Pull request by Sekhmet**
## Summary

This PR adds markdown support to proposal body. You can now display preview of your proposal in editor as well.

Once submitted proposal will be displayed with Markdown parsed.

Closes: https://github.com/snapshot-labs/sx-ui/issues/522

## Test plan

- Create proposal with Markdown
- View this proposal.

### CODE REVIEW

File: src/components/Markdown.vue [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+const parsed = computed(() => remarkable.render(props.body));
+</script>
+
+<template>
+  <div class="markdown-body break-words" v-html="parsed" />
+</template>
+
+<style lang="scss">
+.markdown-body {
```
Comment (Sekhmet) [inline src/components/Markdown.vue:24]: Styles from Snapshot.

**Review by j0hnfl0w — APPROVED** [review]
tACK

**Comment by bonustrack** [comment]
Can you force to have only the styles specified in the issue?

File: src/views/Editor.vue [hunk 2]
```diff
@@ -159,7 +160,22 @@ watch(proposalData, () => {
           title: 'Title'
         }"
       />
-      <div class="s-base mb-5">
+      <div class="flex">
+        <Link
+          :is-active="!previewEnabled"
+          text="Editor"
```
Comment (bonustrack) [inline src/views/Editor.vue:166]: How about using same naming than GitHub? Editor is the whole page not just the body
```suggestion
          text="Write"
```

**Comment by bonustrack** [comment]
@Sekhmet Correct

**Comment by Sekhmet** [comment]
@bonustrack doesn't seem possible with remarkable, we can enable global features like headings, but not specific ones:
I could enable:
- headings (# - ######)
- links (both links and images)
- emphasis (bold and itallic)
- blockquote

**Comment by bonustrack** [comment]
@Sekhmet Ok sound good, maybe just make it so h1 and h2 use the same font-size in the proposal body. I noticed that `remarkable` seem to not be that popular anymore, was not updated for quite some time we may want to replace it with another one at some point.

**Pull request merged**